pkg/config/app/kumactl/v1alpha1/config_helpers.go

```diff
@@ -56,13 +60,14 @@ func (cfg *Configuration) GetControlPlane(name string) (int, *ControlPlane) {
 	return -1, nil
 }
 
-func (cfg *Configuration) AddControlPlane(cp *ControlPlane) bool {
+func (cfg *Configuration) AddControlPlane(cp *ControlPlane) error {
```

Comment:
Could we revert to the original version with `bool` return type ?

So that all methods inside `config_helpers.go` consistently return `bool`

Reply:
Fixed